platform/iphone/export/export.cpp

```diff
@@ -1370,6 +1384,25 @@ Error EditorExportPlatformIOS::_export_ios_plugins(const Ref<EditorExportPreset>
 
 		p_config_data.cpp_code += plugin_cpp_code.format(plugin_format, "$_");
 	}
+
+	// Update Linker Flag Values
+	String result_linker_flags = " ";
+	for (int i = 0; i < plugin_linker_flags.size(); i++) {
+		String flag = plugin_linker_flags[i];
```

Comment:
fairly minor thing, this could be a const& so we don't copy the string, probably doesn't matter too much though as this code is rarely called (still good practice though).

Reply:
I guess this could later be done in other places in `export.cpp`